forta-uniswap-event-py/src/agent.py

```diff
@@ -0,0 +1,216 @@
+import binascii
+import json
+
+from forta_agent import Finding, FindingType, FindingSeverity
+from web3 import Web3
+import web3
+
+# Uniswap v2: Router 2
+# 0x7a250d5630b4cf539739df2c5dacb4c659f2488d
+ROUTER_ADDR = Web3.toChecksumAddress("0x7a250d5630b4cf539739df2c5dacb4c659f2488d")
+
+ETHER_THRESHOLD = Web3.toWei("5", "ether")
+
+CONTRACT_INST = None
+
+
+class AttrDict(dict):
+    """
+    Allows a dictionary to act as a class object and attributes can be retrieved with
+    the . "dot" notation or the "['']" square bracket notation. This is a very helpful helper
+    class since web3 typically uses square bracket notation and forta-agent uses dot syntax
```

Comment:
Uber Nit: `since` should be used when there is a temporal relationship, `because` should be used otherwise.
```suggestion
    class because web3 typically uses square bracket notation while forta-agent uses dot syntax.
```

Reply:
Actually did not know that was the differentiating factor between them, super cool